content/cpp/concepts/math-functions/terms/fmod/fmod.md

```diff
@@ -0,0 +1,94 @@
+---
+Title: 'fmod()'
+Description: 'Computes the floating point remainder of numerator/denominator (rounded towards zero)'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`fmod()`** function in C++ computes the floating point remainder of numerator/denominator (rounded towards zero).
+
+## Syntax
+
+```pseudo
+fmod (x, y)
+```
+
+x: The value of numerator,
+y: The value of denominator.
+
+## Prototype [C++11 Standard]
+
+```cpp
+double fmod(double x, double y);
+float fmod(float x, float y);
+long double fmod(long double x, long double y);
+// Additional overloads
+double fmod(Type1 x, Type2 y);
+```
+
+## Definition and Return Type
+
+Additional overloads are provided in this header (<cmath>) for other combinations of arithmetic types (Type1 and Type2): These overloads effectively cast its arguments to double before calculations, except if at least one of the arguments is of type long double (in which case both are casted to long double instead).
+
+If successful, returns the floating-point remainder of the division x/y as defined above.
```

Comment:
Consider this statement instead:

"If the computation is successful, a floating-point remainder of the division  of `x` by `y` is returned as defined above."